libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigation.java

```diff
@@ -1162,4 +1163,8 @@ private static String obtainUserAgent(MapboxNavigationOptions options) {
       return MAPBOX_NAVIGATION_USER_AGENT_BASE + BuildConfig.MAPBOX_NAVIGATION_VERSION_NAME;
     }
   }
+
+  public boolean isFreeDriveEnabled() {
```

Comment:
Do we want to expose this API publicly? It's used here https://github.com/mapbox/mapbox-navigation-android/blob/eea408ca1adf6019b1b6a8d4dd9beb8c7831ba61/app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/ComponentNavigationActivity.java#L344 but it seems that the client already knows when free drive is enabled / disabled. What do you think?

Reply:
to check it, we can add a local flag and change it after
```navigation.enableFreeDrive();``` or ```navigation.disableFreeDrive();```

but I think it's easier to add ```public boolean isFreeDriveEnabled()``` and use it

Reply:
Yeah, I understand is easier but the questions here are do we want to expose a new public API for this? Is it worth it? Also, are we "duplicating" behavior somehow (as we're exposing enable / disable APIs) and clients can tell easily?

Reply:
ok, let's safe public API, I will use a bool flag inside Activity